3 - Model fitting and hypothesis testing.Rmd

```diff
@@ -0,0 +1,152 @@
+---
+title: "3 - Model fitting and hypothesis testing"
+author: "Joshua Soon"
+date: '25 August 2022'
+output: html_document
+editor_options: 
+  chunk_output_type: console
+---
+
+
+
+Default dependencies here and yaml are required for config.yml parsing to obtain parameters.
+```{r echo=FALSE, warning=FALSE, message=FALSE}
+library(here)
+library(yaml)
+```
+
+
+
+Load the packages for this specific Rmd notebook.
+```{r echo=FALSE, warning=FALSE, message=FALSE}
+library(DESeq2)
+library(apeglm)
+library(dplyr)
+library(tidyverse)
+```
+
+
+
+Run differential expression testing
+```{r echo=FALSE, warning=FALSE, message=FALSE}
+RNA_seq_dds <- DESeq(RNA_seq_dds)
+```
+
+
+
+Plot dispersion estimates
+```{r echo=FALSE, warning=FALSE, message=FALSE}
+plotDispEsts(RNA_seq_dds)
+```
+
+
+
+Define contrasts, extract results table, and shrink the log2 fold changes, specify groups for comparison
+```{r echo=FALSE, warning=FALSE, message=FALSE}
+RNA_seq_res_table_unshrunken <- results(RNA_seq_dds,
```

Comment:
Actually for this repository and the original, I was wondering if it would be better to add in the argument ` independentFiltering = TRUE`, since this may need to be toggled to `FALSE` if cpm_filtering is set? You can also mention this in the markdown above the code.

Reply:
Ok, done!